scripts/setup-dependencies

```diff
@@ -18,4 +18,5 @@ pip install -r core/requirements_test.txt
 
 pip install -e core/
 pip install ulid-transform # this is in Adaptive-lighting's manifest.json
+pip install aiohue
```

Comment:
Does this mean your custom package is no longer needed? :)

Reply:
Yes, I use the V2 API.
I had to overwrite a class because there was a serialization issue but it works